Se agrego información a la página de Acerca de Nosotros, se realizo u…

GermanRusco opened the pull request:
…na clase para las imagenes de quienes somos y se agrego información de cada uno de los integrantes.

GermanRusco committed:
Se agrego información a la página de Acerca de Nosotros, se realizo una clase para las imagenes de quienes somos y se agrego información de cada uno de los integrantes.

priscilabgmoya commented on page/quienesSomos.html:
```diff
@@ -14,8 +14,42 @@
+        <p>Somos un grupo de programadores con el sueño de poder completar la pokedex con los mas de 600 pokemons que existen en este mundo.</p>
+        <hr>
+        <section class="d-flex">
+            <article class="card col-lg-6 m-3">
```
en este caso, faltaría que se active la col, es decir, col col-xl-4 col-sm-12 col-md-6

priscilabgmoya committed:
Update styles.css

agrego modificaciones pedidas por mi compañero german

priscilabgmoya committed:
Update quienesSomos.html

agrego la parte responsive de la pagina quienes somos, modificaciones pedidas por mi compañero german

priscilabgmoya reviewed (approved):
pagina quienes somos modificada y siendo responsive
